src/app/shared/components/category-select/category-select.component.scss

```diff
@@ -18,6 +20,10 @@
   font-family: Roboto, "Helvetica Neue", sans-serif;
 }
 
+::ng-deep .dropdown-panel {
```

Comment:
needs to be isolated, styles under ng-deep could be applied to .dropdown-panel class across the app

Reply:
unified by an additional class "direction"